scss/_variables.scss

```
@@ -1078,6 +1078,10 @@ $btn-social-networks: (
   "tiktok": (
     "color": #ff2c55,
     "icon": "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 32 32'><path d='M7.024 30.054C4.584 28.212 3 25.235 3 21.876c0-5.59 4.39-10.123 9.805-10.123.45 0 .899.031 1.345.094v5.6a4.363 4.363 0 0 0-1.361-.218c-2.477 0-4.485 2.074-4.485 4.631 0 1.809 1.003 3.374 2.467 4.137l.31.146a4.348 4.348 0 0 0 1.708.348c2.471 0 4.476-2.065 4.484-4.615V0h5.335v.704c.02.211.046.42.082.63l.08.404a7.668 7.668 0 0 0 3.306 4.769A7.22 7.22 0 0 0 30 7.665V8.83l-.199-.047-.182-.047.381.094v4.312a12.4 12.4 0 0 1-7.392-2.443v11.177c0 5.591-4.39 10.124-9.804 10.124-2.02 0-3.898-.63-5.458-1.712l-.322-.234Z'/></svg>"
+  ),
+  "x": (
+    "color": #1da1f2,
+    "icon": "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 20 19'><path d='M0.0486667 0L7.77044 10.3247L0 18.7191H1.74889L8.552 11.3696L14.0487 18.7191H20L11.8438 7.81355L19.0764 0H17.3276L11.0624 6.76867L6.00022 0H0.0486667ZM2.62067 1.28822H5.35467L17.4278 17.4307H14.6938L2.62067 1.28822Z'/></svg>"
```

Just ran svgo on this: 
```suggestion
    "icon": "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 20 19'><path d='M.049 0 7.77 10.325 0 18.719h1.749l6.803-7.35 5.497 7.35H20L11.844 7.814 19.076 0h-1.748l-6.266 6.769L6 0H.05ZM2.62 1.288h2.734l12.073 16.143h-2.734L2.62 1.288Z'/></svg>"
```

And just used the logo from [their documentation](https://about.x.com/en/who-we-are/brand-toolkit): 
```suggestion
    "icon": "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 1200 1227'><path d='M714.163 519.284 1160.89 0h-105.86L667.137 450.887 357.328 0H0l468.492 681.821L0 1226.37h105.866l409.625-476.152 327.181 476.152H1200L714.137 519.284h.026ZM569.165 687.828l-47.468-67.894-377.686-540.24h162.604l304.797 435.991 47.468 67.894 396.2 566.721H892.476L569.165 687.854v-.026Z'/></svg>"
```

---

We have to use the version of the logo defined in our design kit.